test/setup/tools/index.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  path as skopeoPath,
```

Comment:
why not ```import * as skopeo from './skopeo'``` if you want the skopeo prefix?
your import will be 1 line long
and usage will be skopeo.path instead of skopeopath